assets/resources/scss/_navMenu.scss

```diff
@@ -56,7 +104,8 @@ header {
   &__btnText {
     font-family: 'Montserrat-Bold';
     font-weight: bold;
-    font-size: 0.8rem;
+    font-size: 0.75rem;
+    letter-spacing: 2px;
```

Comment:
rem? font size is in rem so this should be too

Reply:
Fixed.